tests/functional/c/class_protocol_ellipsis.py

```diff
@@ -0,0 +1,41 @@
+""""Tests for return type checkers for protocol methods with ellipsis function body"""
+# pylint: disable=missing-class-docstring
+from typing import Any, Iterator
+
+
+class MyClass:
```

Comment:
Can you add a comment above that line with:
* The link to the Github issue
* Mentioning that `invalid-*-returned` warnings should **not** be emitted for stub functions.